src/Sulu/Bundle/MediaBundle/Domain/Event/MediaPreviewImageCreatedEvent.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\MediaBundle\Admin\MediaAdmin;
+use Sulu\Bundle\MediaBundle\Entity\Collection;
+use Sulu\Bundle\MediaBundle\Entity\FileVersionMeta;
+use Sulu\Bundle\MediaBundle\Entity\MediaInterface;
+
+class MediaPreviewImageCreatedEvent extends DomainEvent
```

Comment:
think we should caa this `MediaPreviewImageAddedEvent` because the media that is added as preview image might have already existed before

Reply:
or maybe just use the `MediaPreviewImageModifiedEvent` and set `oldPreviewImageId` to null? i think that would be even simpler

Reply:
Then I would rename this to `MediaPreviewImageChangedEvent`, because it's not a modification of the preview image itself, it's really just changing the preview image

Reply:
@nnatter WDYT?

Reply:
yes sounds good to me! 🙂

Reply:
not sure about this anymore. i am afraid that it will get confusing when to use `Modify` vs `Change` because they basically mean the same thing 🙈  but we can discuss this in https://github.com/sulu/sulu/pull/5939/files#r624992008